libraries/matware/jupgrade/schemas/1.5/categories.php

```diff
@@ -188,6 +188,16 @@ public function dataHook($rows = null)
 			$this->_step->_nextID($total);
 		}
 
+		// rebuild table
```

Comment:
JTable->rebuild() is already running at the end of each step https://github.com/fastslack/matware-libraries/blob/master/libraries/matware/jupgrade/category.php#L127